cpp/velox/compute/VeloxExecutionCtx.cc

```diff
@@ -259,4 +259,52 @@ void VeloxExecutionCtx::releaseColumnarBatchSerializer(ResourceHandle handle) {
   columnarBatchSerializerHolder_.erase(handle);
 }
 
+int32_t VeloxExecutionCtx::getColumnarBatchPerRowSize(std::shared_ptr<ColumnarBatch> cb) {
+  auto veloxBatch = std::dynamic_pointer_cast<VeloxColumnarBatch>(cb);
+  VELOX_DCHECK(veloxBatch != nullptr, "getColumnarBatchPerRowSize batch should be VeloxColumnarBatch");
+  auto rowVector = veloxBatch->getFlattenedRowVector();
+  auto fast = std::make_unique<facebook::velox::row::UnsafeRowFast>(rowVector);
+  return fast->rowSize(0);
+}
```

Comment:
why not put this method to ColumnarBatch class? it will make ExecutionCtx more expansion. cc @zhztheplayer

Reply:
@Yohahaha Make sense to me. I will move this method to ColumnarBatch later. Thanks again.